### PR TITLE
Twu/search android

### LINKED ISSUE #254: Link search code

### PR DESCRIPTION
**Pull request by twuster**
fixes #254 and #226

### CODE REVIEW

**Comment by twuster** [comment]
Do this one first. It's ready... like my body.

File: PHC-Android/PHC/src/main/java/phc/android/Checkin/PersonalInfoFragment.java [hunk 1]
```diff
@@ -270,14 +270,6 @@ private void prepopulateForm() {
             }
         }
 
-        if (!ethnicity.equals("null")) {
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/java/phc/android/Checkin/PersonalInfoFragment.java]: wait why did you delete the ethnicity spinner?

File: PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java [hunk 2]
```diff
@@ -69,19 +69,19 @@
     // Shared Preferences
     private SharedPreferences mUserPreferences;
     private ProgressDialog mProgressDialog;
-    /** Parent Activity **/
+    // Parent Activity *
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java]: also i noticed that we seem to be using opposite conventions for documentations.
@chenhoward preferred the /** **/ for documenting class variables, and the "//" for in-line comments within methods.

File: PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java [hunk 3]
```diff
@@ -274,129 +338,55 @@ public void onErrorResponse(VolleyError volleyError) {
...
+            SharedPreferences.Editor editor = sharedPreferences.edit();
+            try {
+                editor.putString("SS_Num", jsonObject.getString("SS_Num__c"));
+                editor.putString("FirstName", jsonObject.getString("FirstName"));
+                editor.putString("LastName", jsonObject.getString("LastName"));
+                editor.putString("Phone", jsonObject.getString("Phone"));
+                editor.putString("Birthdate", jsonObject.getString("Birthdate__c"));
+                editor.putString("Email", jsonObject.getString("PersonEmail"));
+                editor.putString("Gender", jsonObject.getString("Gender__c"));
+                editor.putString("Language", jsonObject.getString("Primary_Language__c"));
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java]: you'll also need to include Ethnicity Spinner, and the LGBT, Foster Care, and Veteran checkboxes

File: PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java (hunk 2 above)
Comment (twuster) [inline PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java]: I'm pretty sure  // is better.  /** **/ just deems too messy. I've been changing them to //

File: PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java (hunk 3 above)
Comment (twuster) [inline PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java]: I'm going to add them all in another  pull request
